core/src/components/radio/test/states/radio.e2e.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from '@playwright/test';
+import { test } from '@utils/test/playwright';
+
+test.describe('radio: states', () => {
+  test('should render disabled radio correctly', async ({ page, skip }) => {
```

Comment:
This is another one that would be good to have checked by default to show `ios` mode better.